Chapter02/recipe-0011/README.md

```diff
@@ -0,0 +1,100 @@
+# Discovering the operating system
+
+CMake is a set of cross-platform tools. Nevertheless it can be very useful to
+know on which operating system (OS) the configure or build step is executed:
+either to tweak CMake code for a particular OS, to enable conditional
+compilation depending on the OS or to use compiler-specific extensions if
+available. In this recipe we will demonstrate how to use CMake to detect the
+OS.
+
+
+## Ingredients
+
+- A C++ compiler
+- CMake (any version higher than 3.0)
```

Comment:
This is a general requirement for the book itself. It's redundant here (and in the following recipes)

Reply:
OK thanks - will remove. Got inspired by one of your recipes and thought it was required but maybe it was just the first one.